fennel/datasets/test_invalid_dataset.py

```diff
@@ -962,7 +962,8 @@ def location_ds(cls, location: Dataset):
 
     with pytest.raises(Exception) as e:
         client.log("fennel_webhook", "mysql_relayrides.location", df)
+    print(str(e.value))
     assert (
         str(e.value)
-        == "Error while executing pipeline `location_ds` in dataset `LocationDS`: Error in assign node for column `latitude_int` for pipeline `LocationDS2.location_ds`, cannot safely cast non-equivalent float64 to int64"
+        == "Error while executing pipeline `location_ds` in dataset `LocationDS`: Error in assign node for column `latitude_int` for pipeline `LocationDS2.location_ds`, Field `latitude_int` is of type int, but the column in the dataframe is of type `Float64`. Error found during checking schema for `LocationDS2.location_ds`."
```

Comment:
The last sentence `...Error round during checking schema...` feels quite redundant? Could you please remove that from wherever the error is generated?